lib/common.bash

```diff
@@ -155,19 +158,26 @@ check_processes() {
 	done
 }
 
-# Checks that pods were not left
-check_pods() {
-	if [ -d ${VC_POD_DIR} ]; then
+# Checks that pods were not left in a directory
+check_pods_in_dir() {
+    local DIR=$1
+    if [ -d ${DIR} ]; then
 		# Verify that pods were not left
-		pods_number=$(ls ${VC_POD_DIR} | wc -l)
+		pods_number=$(ls ${DIR} | wc -l)
 		if [ ${pods_number} -ne 0 ]; then
-			die "${pods_number} pods left and found at ${VC_POD_DIR}"
+            ls ${DIR}
+			die "${pods_number} pods left and found at ${DIR}"
 		fi
 	else
-		echo "Not ${VC_POD_DIR} directory found"
+		echo "Not ${DIR} directory found"
 	fi
 }
 
+# Checks that pods were not left
+check_pods() {
+	check_pods_in_dir ${VC_POD_DIR}
```

Comment:
So, I reckon we should add the SBS check here as well... then it will check sbs after every test, and not just vm template tests.... as nothing should ever leave a dirty sbs dir, should it?

Reply:
> So, I reckon we should add the SBS check here as well... then it will check sbs after every test, and not just vm template tests.... as nothing should ever leave a dirty sbs dir, should it?

@grahamwhaley If we want to do that.  what I think update all the check function to check_sanity.sh and call in after each test in Makefile.

Reply:
Sure - and I'm happy if we do that on a different PR - as I suspect it will break under the CI right now and may take some time to find/fix all the issues it shows - and I don't think we should have to hold up this PR .